app/build.gradle.kts

```diff
@@ -248,6 +248,7 @@ dependencies {
     implementation(libs.androidx.credentials)
     implementation(libs.google.hilt.android)
     ksp(libs.google.hilt.compiler)
+    implementation(libs.google.play.review)
```

Comment:
I think we need to declare this as a `standardImplementation` dependency so that it doesn't get included in F-Droid builds. That means we'll need `Standard` and `Fdroid` implementations of `AppReviewManager` in their respective source dirs.